static/js/components/NewInstrument.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Form from "@rjsf/material-ui";
+import { showNotification } from "baselayer/components/Notifications";
+import { submitInstrument } from "../ducks/instrument";
+
+const NewInstrument = () => {
+  const { instrumentList } = useSelector((state) => state.instruments);
+  const { telescopeList } = useSelector((state) => state.telescopes);
+  const dispatch = useDispatch();
+
+  const handleSubmit = async ({ formData }) => {
+    if (formData.group_id === -1) {
+      delete formData.group_id;
+    }
+    const result = await dispatch(submitInstrument(formData));
+    if (result.status === "success") {
+      dispatch(showNotification("Instrument saved"));
+    }
+  };
+
+  const api_classnames = [{ enum: [null], title: "No API" }];
```

Comment:
I'm getting a weird warning in the console: 
`Warning: `value` prop on `input` should not be null.`
I am guessing it has to do with putting null values for the default API class, or something. Worth looking into it.

Reply:
yes made this just "", so I think better